fastjmd95/jmd95wrapper.py

```diff
@@ -0,0 +1,47 @@
+
+import numpy as np
+import dask.array as dsa
+import dask
+import xarray as xr
+
+import fastjmd95.jmd95numba as jmd95numba
+
+
+def rho_x(s, t, p):
+    if any([isinstance(s,dask.array.core.Array),
+           isinstance(t,dask.array.core.Array),
+           isinstance(p,dask.array.core.Array)]):
```

Comment:
Since we call this function three, times, should we instead define a function with the signature:
```python
any_dask_array(*args)
```
?